File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-javadsl/c/src-c/main/java/impl/BazServiceImpl.java
package impl;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import java.util.concurrent.CompletableFuture;
import javax.inject.Inject;
import api.BazService;

import akka.stream.javadsl.Source;

public class BazServiceImpl implements BazService {

  @Override
  public ServiceCall<NotUsed, String> baz() {
    return request -> CompletableFuture.completedFuture("ack baz");
  }
}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-scaladsl/c/src-c/main/scala/impl/BazServiceImpl.scala
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package impl

import akka.NotUsed
import com.lightbend.lagom.scaladsl.api.ServiceCall
import scala.concurrent.Future
import api.BazService

class BazServiceImpl extends BazService {
  override def baz = ServiceCall { _ =>
    Future.successful("ack baz")
  }
}
